app/views/pages/developer-guide/records/crud-records-using-ajax.liquid

```diff
@@ -131,7 +131,7 @@ First, you'll need a GraphQL query to pull out the data:
 
 ```graphql
 query feedback($per_page: Int = 10) {
-  models(
+  records(
     filter: {
       name: { value: "feedback" }
```

Comment:
@diana-lakatos in this context `name` is even older name for `model_schema_name` and needs to be changed to `table`

Reply:
Edited, thank you!